lib/step.rb

```diff
@@ -120,9 +134,33 @@ def run(dry_run: false, fail_fast: false)
 
         if @verbose
           puts
-        elsif failed?
+        elsif !passed?
           puts @output
           puts
+
+          [@named_args].flatten.each do |name|
+            next if name.blank?
+
+            path, line = begin
```

Comment:
Is line going to be nil in every case or am I reading something wrong?

Reply:
No, `line` is a number when this block returns `method_location`.

Reply:
Ah, you're right. The first element of `method_location` is a `String`, so that will never be equal to `formula.path`. I need a `#to_s` here; thanks.